README.md

```diff
@@ -187,6 +189,11 @@ where:
 - `--p-email` is your email address (required by NCBI)
 - `--output-dir` directory where the downloaded metadata, sequences and IDs for failed downloads are stored as QIIME 2 artifacts
 
+## Downstream analysis
+For more information on how to use q2-fondue outputs within the QIIME 2 ecosystem see section [Downstream analysis in QIIME 2](./tutorial/tutorial.md#downstream-analysis-in-qiime-2) in the tutorial.     
+
+To learn more on how to prepare q2-fondue outputs for further analysis outside of QIIME 2 see tutorial section [Prepare downstream analysis outside of QIIME 2](./tutorial/tutorial.md#prepare-downstream-analysis-outside-of-qiime-2). 
```

Comment:
This section/sentence should perhaps be preceded with an acknowledgement that... "Some downstream analyses may need to rely on tools outside of QIIME 2. q2-fondue can be used for diverse applications, and data can be exported directly into FASTQ and other interoperable formats. Note that the exported files will no longer contain integrated provenance information (which is unique to QIIME 2 Artifacts), but this metadata can be exported also and the original artifacts will retain the provenance data for traceability purposes."

Reply:
I like this. Your suggestion is included with a slight edit in the second sentence.